lib/rubocop/cop/lint/ambiguous_operator.rb

```diff
@@ -60,7 +60,8 @@ def find_offense_node_by(diagnostic)
           end
 
           ast.each_node(:send).find do |send_node|
-            offense_position?(send_node.first_argument, diagnostic) &&
+            send_node.first_argument &&
```

Comment:
We can reduce a bit of repetition by assigning `send_node.first_argument` to a variable.